### PR TITLE
Issue 334 doxygen

### PR DESCRIPTION
**Pull request by mortenjc**
### Issue reference / description

Update doxygen headers and copyright headers in src/*/ only one level deep.

One file removed which was no longer used but else no code changes.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

## Nominate for Group Code Review (Anyone can nominate it)
Indicate if you think the code should be reviewed in a Thursday code review session.

- [ ] Recommend for group code review

Also, nominate it on the code_review Slack channel.

### CODE REVIEW

File: src/modules/jalousie/SumoMappings.cpp [hunk 1]
```diff
@@ -1,4 +1,11 @@
-/** Copyright (C) 2016, 2017 European Spallation Source ERIC */
+// Copyright (C) 2019-2020 European Spallation Source, see LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief Digital geometry for SUMO (Sub-Modules) - implementation
```
Comment (MortenHilkerSkaaning) [inline src/modules/jalousie/SumoMappings.cpp:6]: SUMO is for SUper MOdule...

File: src/modules/jalousie/SumoMappings.h [hunk 2]
```diff
@@ -1,4 +1,11 @@
-/* Copyright (C) 2017-2018 European Spallation Source, ERIC. See LICENSE file */
+// Copyright (C) 2019-2020 European Spallation Source, see LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief Digital geometry for SUMO (Sub-Modules)
```
Comment (MortenHilkerSkaaning) [inline src/modules/jalousie/SumoMappings.h:6]: SUMO is for SUper MOdule...

File: src/modules/readout/ReadoutParserTest.cpp [hunk 3]
```diff
@@ -1,4 +1,11 @@
-/** Copyright (C) 2016, 2017 European Spallation Source ERIC */
+// Copyright (C) 2017-2020 European Spallation Source, ERIC. See LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief Unit tests for ReadoutPOarser
```
Comment (MortenHilkerSkaaning) [inline src/modules/readout/ReadoutParserTest.cpp:6]: ReadoutParser ?

**Review by MortenHilkerSkaaning — APPROVED** [review]

File: src/common/SPSCFifo.h [hunk 4]
```diff
@@ -1,9 +1,8 @@
-/* Copyright notice see below */
+// Copyright notice see below - author Kjell Hedström, hedstrom@kjellkod.cc
```
[the remark this answers is not in the document]
Comment (mortenjc) [reply src/common/SPSCFifo.h:1]: It is already in the file

**Pull request merged**